selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/CodenvyEditor.java

```diff
@@ -514,6 +514,10 @@ public void waitTextInHoverPopUpEqualsTo(String expectedText) {
     }
   }
 
+  public void waitAppearanceHowerPopUp() {
```

Comment:
`waitAppearance` > `waitHoverPopupAppearance`
I would also recommend to reuse it in `waitTextInHoverPopUpEqualsTo()` method as well.

Reply:
We discussed it on meeting and and decided to leave as is. I do not mind rework this but i just want that our decision will be ok for all.

Reply:
Agree.